src/Mod/Part/Gui/DlgPrimitives.cpp

```diff
@@ -111,6 +111,11 @@ const char* gce_ErrorStatusText(gce_ErrorType et)
     }
 }
 
+static QString sb_Quantity(Gui::QuantitySpinBox *qs)
```

Comment:
Can we name that better? I don't have any idea what `sb_` here means. `safeQuantityQString` maybe?

Reply:
Yeah, feel free to suggest better name (sb stands for SpinBox (Deplhi used that naming scheme in the dark past)). I plan to revisit that file and clean it up.
Please note that previously tried to clean up things on the way, but patchset grow too wide and needed constant rebasing, so I surrended that.

Reply:
`safeQuantityQString` works for me, it gives at least vague idea on what it is doing